Content.Server/Zombies/ZombieSystem.cs

```diff
@@ -380,7 +398,38 @@ private void OnMindAdded(Entity<ZombieComponent> ent, ref MindAddedMessage args)
         // Remove the role when getting cloned, getting gibbed and borged, or leaving the body via any other method.
         private void OnMindRemoved(Entity<ZombieComponent> ent, ref MindRemovedMessage args)
         {
-            _role.MindRemoveRole<ZombieRoleComponent>((args.Mind.Owner,  args.Mind.Comp));
+            _role.MindRemoveRole<ZombieRoleComponent>((args.Mind.Owner, args.Mind.Comp));
         }
+
+        #region Goob Language Changes
+
+        /// <summary>
+        ///     This forces the languages to reset and apply only the current language for the entity based on Zombie Component.
+        /// </summary>
+        private void OnLanguageApply(Entity<ZombieComponent> ent, ref DetermineEntityLanguagesEvent args)
+        {
+            if (ent.Comp.LifeStage is
+                ComponentLifeStage.Removing
+                or ComponentLifeStage.Stopping
+                or ComponentLifeStage.Stopped)
```

Comment:
This reads a little better.
```suggestion
            if (ent.Comp.LifeStage is ComponentLifeStage.Removing
                or ComponentLifeStage.Stopping
                or ComponentLifeStage.Stopped)